src/qibo/backends/clifford_operations.py

```diff
@@ -0,0 +1,436 @@
+import numpy as np
```

Comment:
Same here. Seems like the entire module does not need to be user-facing

Reply:
what do you mean?

Reply:
I think this entire file does not need to be user-facing. `_clifford_operations.py`

Reply:
I cannot rename the gate functions names as they need to be called the same as the standard qibo gates. I could rename the module though.

Reply:
Yeah just the module is enough